client/src/common/components/Chips.scss

```diff
@@ -0,0 +1,34 @@
+@import 'partials';
+
+.chips {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  background: $gray2;
+  width: auto;
+  padding: 0 four-by(3);
+  height: 32px;
+  border-radius: 16px;
+
+  &__icon {
```

Comment:
Personally, I would go for now icon display.

Reply:
I left icon as an option for future use of this component